### DISCUSSION TITLE
Steam Deck setup

### DISCUSSION THREAD

**Discussion by zenseii** [ihhub/fheroes2#6427] [Q&A] [answered]
A user is having issues with where to place the original files on their steam deck:
https://www.reddit.com/r/HoMM/comments/zy152v/homm_ii_fheroes2_on_steam_deck_need_help/

Hi, @Laserlicht, would you have the time to share your expertise in this area?

**Comment by Laserlicht** [comment, marked as answer]
Is resolved. See answer on Reddit.

You can now use the GOG-EXE directly.

**Reply by zenseii** [reply]
Thank you for looking into this so quickly and for continuing to provide support!
